deployer/src/deployer/search/__init__.py

```diff
@@ -21,7 +21,7 @@ def index(
 ):
     # We can confidently use a single host here because we're not searching
     # a cluster.
-    connections.create_connection(hosts=[url])
+    connections.create_connection(hosts=[url], retry_on_timeout=True)
```

Comment:
Turns out, this has nothing to do with the bulk indexing. 
But it feels like quite a nice feature to turn on. Why wouldn't you want to have this on? If it can save the bacon with a bit of retrying, isn't that eons better than having to wait till tomorrows cronjob?